src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenShortCircuitOperatorTests.cs

```diff
@@ -7443,12 +7444,13 @@ public void ConditionalAccessOffConstrainedTypeParameter_Field()
         {
             var source = @"
 using System;
+using System.Globalization;
 
 class Program
 {
     static void Main(string[] args)
     {
-        var obj1 = new MyObject1 { MyDate = DateTime.Parse(""2017-11-13T14:25:00Z"") };
+        var obj1 = new MyObject1 { MyDate = DateTime.Parse(""2017-11-13T14:25:00Z"", CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal) };
```

Comment:
>DateTime.Parse(""2017-11-13T14:25:00Z"", CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal) [](start = 44, length = 104)

Can we simply use DateTime(Int64) constructor that takes the original number of tiks (636461511000000000)?

Reply:
I don't see why not. If that's preferable, I can update the PR.

Reply:
> I don't see why not. If that's preferable, I can update the PR.

I think that would be preferable over making the unit-test dependent on Globalization APIs.